generators/aws-containers/lib/cf.js

```diff
@@ -0,0 +1,297 @@
+const AWS = require('aws-sdk'); // eslint-disable-line
+const _ = require('lodash');
+const chalk = require('chalk');
+
+const STACK_LISTENER_INTERVAL = 15000;
+const PHYSICAL_RESOURCE_SEPARATOR = ':';
+const STACK_EVENT_STATUS_DISPLAY_LENGTH = 35;
+
+let stdOut = message => console.error(message.trim()); // eslint-disable-line
+let stdErr = message => console.error(message.trim()); // eslint-disable-line
+module.exports = class CF {
```

Comment:
Is it okay to rename the class to `CloudFormation ` instead of using an abbreviation?

Reply:
We've named it `cf` in order to be consistant with the other classes in `lib/` : `ecr` and `ssm`